Sources/Basic/OutputByteStream.swift

```diff
@@ -48,62 +50,94 @@ public class OutputByteStream: OutputStream {
         self.buffer = []
     }
 
+    var bufferSize: Int {
+        return 1024
+    }
+
     // MARK: Data Access API
 
     /// The current offset within the output stream.
     public var position: Int {
         return buffer.count
     }
 
-    /// The contents of the output stream.
-    ///
-    /// This method implicitly flushes the stream.
-    public var bytes: ByteString {
-        flush()
-        return ByteString(self.buffer)
-    }
-    
     // MARK: Data Output API
 
     public func flush() {
-        // Do nothing.
+        writeImpl(buffer)
```

Comment:
I think there probably should be separate functions for `writeImpl` and `flushImpl`. The `writeImpl` should be just to copy the bytes from the buffer to the underlying storage as efficiently as possible, but need not imply a flush.